conf/default-core.yaml

```diff
@@ -3,4 +3,12 @@ core:
   minimize: false
   dist: '{{CWD}}/.dist'
   assetPath: '{{CWD}}'
+  configPath: '{{CWD}}/conf'
```

Comment:
what is this for?  it will only be conf for internal clientkit, will be clientkit for all other projects